galaxy.yml

```diff
@@ -1,6 +1,8 @@
 ---
 authors:
   - Ansible Network Community (ansible-network)
+dependencies:
+  "ansible.utils": "*"
```

Comment:
If this is now a hard dependency, we need to update our jobs a bit.  Right now we have to define the dependency in zuul too,  so any reference for build-ansible-collections, that include ansible.netcommon, also need to add ansible.utils.

EG: https://github.com/ansible/ansible-zuul-jobs/blob/master/zuul.d/project-templates.yaml#L41

Once we get a buildset galaxy server in place, we don't need to do this.

Reply:
Yes, we plan to have `ansible.utils` as the base collection which the `ansible.netcommon` collection will depend upon.